vipermonkey/core/expressions.py

```diff
@@ -3688,21 +3719,23 @@ def eval(self, context, params=None):
                 # Call function.
                 #print "WHERE: 5"
                 r = f.eval(context=context, params=params)                        
+                
                         
                 # Set the values of the arguments passed as ByRef parameters.
                 #print "WHERE: 6"
                 if (hasattr(f, "byref_params")):
                     for byref_param_info in f.byref_params.keys():
                         try:
-                            arg_var_name = safe_str_convert(self.params[byref_param_info[1]])
+                            # we can pass arguments with () at the end so we should remove it
+                            arg_var_name = safe_str_convert(self.params[byref_param_info[1]]).replace("(","").replace(")","").replace("'","").replace("'","")
```

Comment:
If you wrap this expression in braces `()` you can place each method call on it's own line, making it less long and more readable:
``` python
arg_var_name = (
    safe_str_convert(self.params[byref_param_info[1]])
    .replace("(", "")
    .replace(")", "")
    .replace("'", "")
    .replace("'", "")
)
```